### ISSUE TITLE
Versions of all packages

### ISSUE THREAD

**Issue by umariqbal-nvidia** [shubham-goel/4D-Humans#31]
Hi, 

Thanks a lot for the great work. Is it possible to provide a detailed version of your conda env with versions for all packages?  My environment created using the provided environment.yml causes some pickling errors in the web dataset. I think it is because some packages are not consistent with your installation. 

Thanks,
Umar

**Comment by carlosedubarreto** [comment]
Here is a list with the installation I did here and is working for me.:


absl-py==1.4.0
aiohttp==3.8.5
aiosignal==1.3.1
antlr4-python3-runtime==4.9.3
async-timeout==4.0.2
attrs==23.1.0
av==10.0.0
black==23.7.0
braceexpand==0.1.7
cachetools==5.3.1
certifi==2022.12.7
charset-normalizer==2.1.1
chumpy==0.70
click==8.1.6
cloudpickle==2.2.1
colorama==0.4.6
contourpy==1.1.0
cycler==0.11.0
Cython==3.0.0
detectron2 @ detectron2-0.6-cp310-cp310-win_amd64.whl
dill==0.3.7
einops==0.6.1
filelock==3.12.2
fonttools==4.42.0
freetype-py==2.4.0
frozenlist==1.4.0
fsspec==2023.6.0
fvcore==0.1.5.post20221221
google-auth==2.22.0
google-auth-oauthlib==1.0.0
grpcio==1.56.2
huggingface-hub==0.16.4
hydra-core==1.3.2
idna==3.4
imageio==2.31.1
iopath==0.1.9
joblib==1.3.1
kiwisolver==1.4.4
lazy_loader==0.3
lightning-utilities==0.9.0
Markdown==3.4.4
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib==3.7.2
mdurl==0.1.2
multidict==6.0.4
mypy-extensions==1.0.0
networkx==3.1
ninja==1.11.1
numpy==1.23.0
oauthlib==3.2.2
omegaconf==2.3.0
opencv-python==4.8.0.74
packaging==23.1
pathspec==0.11.2
Pillow==10.0.0
platformdirs==3.10.0
portalocker==2.7.0
protobuf==4.23.4
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycocotools @ file:////pycocotools-2.0-cp310-cp310-win_amd64.whl
pyglet==2.0.9
Pygments==2.16.1
PyOpenGL==3.1.0
pyparsing==3.0.9
pyrender==0.1.45
python-dateutil==2.8.2
pytorch-lightning==2.0.6
pytube==15.0.0
PyWavelets==1.4.1
pywin32==305
PyYAML==6.0.1
requests==2.28.1
requests-oauthlib==1.3.1
rich==13.5.2
rsa==4.9
safetensors==0.3.1
scenedetect==0.6.2
scikit-image==0.21.0
scikit-learn==1.3.0
scipy==1.10.0
six==1.16.0
smplx==0.1.28
tabulate==0.9.0
tensorboard==2.13.0
tensorboard-data-server==0.7.1
termcolor==2.3.0
threadpoolctl==3.2.0
tifffile==2023.7.18
timm==0.9.5
tomli==2.0.1
torch==1.13.0+cu117
torchmetrics==1.0.2
torchvision==0.14.0+cu117
tqdm==4.65.0
trimesh==3.23.0
typing_extensions==4.7.1
urllib3==1.26.13
webdataset==0.2.48
Werkzeug==2.3.6
yacs==0.1.8
yarl==1.9.2

**Comment by shubham-goel** [comment]
Thanks for responding, @carlosedubarreto ! Assuming @umariqbal-nvidia's issue has been resolved, closing issue.
